src/components/NewChapterForm.tsx

```diff
@@ -0,0 +1,269 @@
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="universityEmail"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>Phone Number</div>
+              <input
+                {...register("phoneNumber")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="phoneNumber"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>College / University</div>
+              <input
+                {...register("university")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="university"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>College / University Address</div>
+              <input
+                {...register("universityAddress")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="universityAddress"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            {/* {title.map((item, index) => (
+            <div key={index} className="w-full">
+              <div>{item}</div>
+              <div>
+                <input className="h-8 w-full rounded-md " />
+              </div>
+            </div>
+          ))} */}
+          </div>
+          <div className=" pt-5">
+            <div>
+              <div>
+                Do you have any experience in student leadership / club
+                organization / storytelling?{" "}
+              </div>
+              <textarea
```

Comment:
`<textarea ... />` should also have `align-top` style to prevent whitespace under.

![image](https://github.com/JumboCode/TheLegacyProject2/assets/74647679/87c77c98-3f7b-405e-b3c1-4dd8903b61e6)